### PR TITLE
Add table of contents

### PR DESCRIPTION
**Pull request by ma-shamshiri**
## Proposed changes
I added a table of contents to make it easier to navigate README.md

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

### CODE REVIEW

**Comment by alexgurr** [comment]
I like this idea, however have a few points:

- It seems this has broken a bunch of the headers - broken
- I'd rather keep the bullet points consistent symbols (•)
- Not a big fan of the rainbow separator. Can we keep it consistent with the others?
- I'd put a `&nbsp;` above the `Table of Contents` headers, to be consistent with other headers

**Comment by alexgurr** [comment]
Thanks for this. Will re-review in the morning and merge it if it's all ok.

**Comment by alexgurr** [comment]
- Slack icon next to the community header still missing

- Arrows are still showing in the table of contents (is there a way around this?)

**Comment by ma-shamshiri** [comment]
Thank you Alex for your comment.
I just added the Slack icon next to the community header.
Also, the arrows have been removed from the table of contents.

File: README.md [hunk 1]
```diff
@@ -1,69 +1,137 @@
...
 
 &nbsp;
-## The Challenges
-### Easy 🙂
+
+<!-- TABLE OF CONTENTS -->
+<h2 id="table-of-contents"> Table of Contents :book:</h2>
+
+<details open="open">
+  <summary>Table of Contents</summary>
```
Comment (alexgurr) [inline README.md:13]:
```suggestion
```

**Review by alexgurr — CHANGES_REQUESTED** [review]

**Comment by alexgurr** [comment]
For some reason the first entry always shows an arrow screen

**Comment by ma-shamshiri** [comment]
Well, the reason the first arrow is displayed is that I used the <details> tag for the table of contents. 
In this case, the user can open and close the table of contents.
Actually <details> tag is basically used to create an interactive widget that one can open and close. 
Do you want me to remove it?

**Comment by alexgurr** [comment]
I don't think it needs to be closable, I think we can just make it static.

**Comment by ma-shamshiri** [comment]
That's right.
I just removed the "details" tag.

**Comment by alexgurr** [comment]
- Everything after the table of contents is inlined (tabbed in) 

&nbsp;
screen

- There's no space/line break between the bullets and afterword

&nbsp;
screen2

**Comment by ma-shamshiri** [comment]
Yes, you are right, sometimes this problem occurs due to improper indentation. 
Is it fine now?

**Comment by alexgurr** [comment]
So I've just realised GitHub provides a table of contents out the box, no custom TOC required. This means we no longer have to keep the TOC updated or use HTML in the readme file. Let me know if there's anything else your PR provides, otherwise I'm closing this for now. 

screengrab

**Pull request closed without merging**